src/hooks/useTxNotifications.ts

```diff
@@ -68,6 +76,44 @@ const useTxNotifications = (): void => {
       unsubFns.forEach((unsub) => unsub())
     }
   }, [dispatch, safeAddress, chain?.shortName])
+
+  /**
+   * Show a notification when a transaction is awaiting confirmation on Safe load
```

Comment:
Thanks for the comment. Please add some more details, e.g. "if there's at least one tx awaiting a confirmation, show a notification for it".